src/AppInstallerCLICore/Commands/ExportCommand.cpp

```diff
@@ -7,6 +7,9 @@
 #include "Workflows/WorkflowBase.h"
 #include "Resources.h"
 
+using AppInstaller::CLI::Workflow::ExecutionStage;
+using AppInstaller::CLI::Workflow::SearchPurpose;
```

Comment:
nit: just `using AppInstaller::CLI::Workflow;` should be enough
And moving inside namespace AppInstaller::CLI to be with existing usings